client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -86,7 +86,9 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
                 justifyContent: 'center',
               }}
             >
-              {t('label.available-quantity', { number: availableQuantity })}
+              {t('label.available-quantity', {
+                number: availableQuantity.toFixed(0),
```

Comment:
Hmm, maybe this should also be rounded, I am surprised that formatter is not used here, @mark-prins should formatter be used everywhere ?

Reply:
Rounded, rather than `toFixed` do you mean? Or the 2 decimal places? ( I suspect the 2dp 🤷 )
I thought that toFixed was better here, because all of the quantity values are integers on this modal - so it keeps the formatting consistent througout.

Reply:
Rounded to a decimal place? 🤔